soak_bench_scripts/soak.sh

```diff
@@ -17,4 +19,4 @@ printf "Starting soak tests...\n"
 
 # Running fromm parsec folder - will need full path
 # This command will loop the test forever until it hits an error
-loop -q target/release/integration_tests-* | tee -a $LOGFILE
+loop -q target/release/integration_tests-* |& tee -a -i --output-error=warn $LOGFILE
```

Comment:
You could also consider adding a timestamp to each line of your log output. There are various ways of doing that:
https://serverfault.com/questions/310098/how-to-add-a-timestamp-to-bash-script-log

It can sometimes be useful to have this when you want to identify stuff that takes a long time to run or things like that.

(Btw, sorry if this comment comes up twice.)

Reply:
Yeah, I had thought about that - good shout, will look to incorporate.